File: app/src/main/java/com/starostinvlad/fan/ViewedScreen/ViewedFragmentContract.java
package com.starostinvlad.fan.ViewedScreen;

import com.starostinvlad.fan.BaseMVP.MvpView;
import com.starostinvlad.fan.GsonModels.News;
import com.starostinvlad.fan.GsonModels.Viewed;

import java.util.List;

public interface ViewedFragmentContract extends MvpView {
    void fillList(List<News> viewedList);

    void showLoading(boolean show);

    void showButton(boolean show);
}

File: app/src/main/java/com/starostinvlad/fan/ViewedScreen/ViewedFragmentContract.kt
package com.starostinvlad.fan.ViewedScreen

import com.starostinvlad.fan.BaseMVP.MvpView
import com.starostinvlad.fan.GsonModels.News
import kotlin.Throws

interface ViewedFragmentContract : MvpView {
    fun fillList(viewedList: List<News?>?)
    fun showLoading(show: Boolean)
    fun showButton(show: Boolean)
}